README.md

```diff
@@ -79,6 +79,17 @@ Kubernetes service and deployment will be created. The service already has a
 Prometheus service-endpoint scraping [configuration](https://raw.githubusercontent.com/prometheus/prometheus/master/documentation/examples/prometheus-kubernetes.yml), Prometheus will pick it up automatically and you can start using the generated
 metrics right away.
 
+### GKE - Google Container Engine
+
+You will need a certified user to deploy on GKE.
+
+```
+gcloud config set container/use_client_certificate True
```

Comment:
Can you either explain in comments or in a paragraph below what each of these commands do? As a user I'd hesitate to execute anything I find in a random github repo :)